pngquant.c

```diff
@@ -75,9 +75,6 @@ use --force to overwrite. See man page for full list of options.\n"
 #include <unistd.h>
 #include <math.h>
 
-extern char *optarg;
```

Comment:
This was added in f8a16a93 for #49, so removing it might cause warnings elsewhere :)